tests/release/pipelines/fbc_release.go

```diff
@@ -53,6 +44,16 @@ var _ = framework.ReleasePipelinesSuiteDescribe("FBC e2e-tests", Label("release-
 	var devFw *framework.Framework
 	var managedFw *framework.Framework
 	var issueId = "bz12345"
+	var fbcApplicationName = "fbc-pipelines-app" + "-" + util.GenerateRandomString(4)
```

Comment:
Although I understand the point of having a separated concatenation of the  `"-"` I don't think it adds much to the readability. I would just add it along with the other part, ie `"fbc-pipelines-app-"`.  Same for the others. WDYT @johnbieren ?.

Reply:
Is that ok to use lower camel case instead?

Reply:
agree with Leandro. He is just saying use `var fbcApplicationName = "fbc-pipelines-app-" + util.GenerateRandomString(4)`

Reply:
The same goes for all the following vars

Reply:
Updated, thanks!